__tests__/erebos-api-bzz-node.js

```diff
@@ -175,4 +175,34 @@ describe('bzz-node', () => {
     expect(dir).toEqual(downloadedDir)
     await fs.remove(tempDirPath)
   })
+
+  it('upload directory of files using uploadDirectoryTar()', async () => {
+    const dir = {
+      'foo.txt': { data: `this is foo.txt - ${uploadContent}` },
+      'bar.txt': { data: `this is bar.txt - ${uploadContent}` },
+    }
+
+    const tempDirPath = path.join(os.tmpdir(), 'erebos-test-temp')
+    await fs.ensureDir(tempDirPath)
+    await fs.outputFile(
```

Comment:
Not an issue but the tests might run a bit faster by writing these 2 files in parallel using `await Promise.all([fs.outputFile(...), fs.outputFile(...)])`